src/js/base/fontawesome.js

```diff
@@ -1,22 +1,14 @@
 import { partial } from 'underscore';
 import Handlebars from 'handlebars/runtime';
-import { findIconDefinition, icon, config } from '@fortawesome/fontawesome-svg-core';
 import '@fortawesome/fontawesome-svg-core/styles.css';
 
-config.autoAddCss = false;
-config.replacementClass = 'svg-inline--fa icon';
-
 function faHelper(prefix, iconName, { hash = {} }) {
-  const faDefinition = findIconDefinition({ prefix, iconName });
-  const faIcon = icon(faDefinition, hash);
+  const svgId = `${ prefix }-fa-${ iconName }`;
+  const svgClass = `fa-${ iconName }`;
+  const svgClasses = hash.classes || '';
+  const svg = `<svg class="icon svg-inline--fa ${ svgClass } ${ svgClasses }"><use xlink:href="#${ svgId }"></use></svg>`;
```

Comment:
Here's how the icon symbols are referenced in a template.  Their ids are based on the naming and those are generate by the FA library